README.md

```diff
@@ -16,32 +16,32 @@
 
 ## Executive summary
 
-- Immuni is a technological solution that centres on **an iOS and Android smartphone app.** It helps us fight the COVID-19 pandemic by notifying users at risk of carrying the virus as early as possible—**even when they are asymptomatic.** These users can then isolate themselves to avoid infecting others, and seek medical advice.
-- Immuni’s design and development are founded on five main principles: **utility, accuracy, scalability, transparency,** and **privacy.**
+- Immuni is a technological solution that centres on **an iOS and Android smartphone app.** It helps us to fight the COVID-19 pandemic by notifying users at risk of carrying the virus as early as possible—**even when they are asymptomatic.** These users should then isolate themselves to avoid infecting others, and seek medical advice.
+- Immuni’s design and development are based on five main principles: **utility, accuracy, scalability, transparency,** and **privacy.**
 - It features a **contact tracing** system based on **Bluetooth Low Energy:**
-  - When two users come sufficiently close to each other for long enough, their devices record each other’s _rolling proximity identifier_ in their local memory. These identifiers are generated from _temporary exposure keys_ and change multiple times an hour. These keys are **generated randomly** and change once per day.
+  - When two users come sufficiently close to each other for long enough, their devices record each other’s _rolling proximity identifier_ in their local memory. These identifiers are generated from _temporary exposure keys_ and change multiple times each hour. These keys are **generated randomly** and change once per day.
```

Comment:
"Each" works as well. We're happy to merga a PR with that change. However, "multiple times an hour" is repeated three times in the document. You may want to change it in all three places, for consistency.